app/services/eyeshade/referral_totals.rb

```diff
@@ -0,0 +1,23 @@
+class Eyeshade::ReferralTotals < Eyeshade::BaseApiClient
+  def perform
+    return perform_offline if Rails.application.secrets[:api_eyeshade_offline]
+
+    response = connection.get do |request|
+      request.headers["Authorization"] = api_authorization_header
+      request.url("/v1/accounts/earnings/referrals/total")
+    end
+
+    JSON.parse(response.body)
```

Comment:
I see a lot of parsing on this data from the eyeshade API. I wonder if it would make more sense to create an ActiveModel in which we initialize it with data from the Eyeshade API and it handles are the parsing. Seems like it would make the code easier to read and more simple.

Reply:
I don't think it's a bad idea, but it's just 3 attributes.

Reply:
Preferably we do this right moving forward by using protocol buffers, https://developers.google.com/protocol-buffers/docs/reference/ruby-generated, but protobuf will give slightly more overhead in development.

Reply:
Yeah, I don't think it's necessary just an idea that we could potentially refactor later